src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -545,21 +553,13 @@ export const MapBlockInternal = ({
 
       featureLayer.bindTooltip(() => {
         if (feature.properties) {
-          const items = Object.entries(feature.properties.dataSets).map(
-            ([dataSetKey, dataSet]) => {
-              const dataSetConfig = dataSetCategoryConfigs[dataSetKey];
-
-              return (
-                `<li>` +
-                `${dataSetConfig.config.label}: ${formatPretty(
-                  dataSet.value,
-                  dataSetConfig.dataSet.indicator.unit,
-                  dataSetConfig.dataSet.indicator.decimalPlaces,
-                )}` +
-                `</li>`
-              );
-            },
+          const dataSetConfig = dataSetCategoryConfigs[selectedDataSetKey];
```

Comment:
Not sure we need this? Couldn't we use `selectedDataSetConfiguration` / `selectedDataSetConfig` instead?

Reply:
yep, sorted